src/scripts/dungeons/Dungeon.ts

```diff
@@ -137,10 +137,14 @@ class Dungeon {
             if (loot.requirement && !loot.requirement.isCompleted()) {
                 return 0;
             }
-            // Minimum of 1 times cleared for division
+            // Minimum of 1 times cleared for division, maximum of 500 times cleared
             const timesCleared = Math.min(500, Math.max(1, App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(this.name)]()));
             // Calculate total weight based on times cleared, minimum weight being original number specified
-            return Math.max(loot.weight, Math.pow(10, loot.weight) / timesCleared) + 1 || 1;
+            if (App.game.challenges.list.hardDungeons.active()) {
+                return Math.max(loot.weight, Math.pow(10, Math.pow(loot.weight, loot.weight)) / timesCleared) + 1 || 1;
+            } else {
+                return Math.max(loot.weight, Math.pow(10, loot.weight) / timesCleared) + 1 || 1;
+            }
```

Comment:
Maybe something like this might work better?
```suggestion
			let power = App.game.challenges.list.hardDungeons.active() ? 20 : 10
			return Math.max(loot.weight, Math.pow(power, loot.weight) / timesCleared) + 1 || 1;
```
10 power:
![image](https://user-images.githubusercontent.com/7288322/147205621-b20a948e-a525-489b-adf7-cb92b5b119fd.png)

15 power:
![image](https://user-images.githubusercontent.com/7288322/147205736-2f880cca-155f-468d-93d1-ee4ec40f76f2.png)

20 power:
![image](https://user-images.githubusercontent.com/7288322/147205638-891cd250-0063-4389-b700-dd81966c7fa8.png)

Reply:
I think 15 is ok.

Reply:
15 or 20 works, but I think I would want to Max timescleared at some low number as well. 1.61% doesn't feel like a big number, but with a big amount of chests per dungeon (especially with no size reduction) it comes out to a fairly frequent rate per run.